### DISCUSSION TITLE
Tailwind CSS Buliding is error

### DISCUSSION THREAD

**Discussion by wolfbaozi** [tailwindlabs/tailwindcss#11841] [Help] [answered]
![image](https://github.com/tailwindlabs/tailwindcss/assets/46016243/c39d7334-5687-4065-b6e0-f74eca1e6cbc)
What is the reason for this problem and how can I solve it, thanks

**Comment by m1her** [comment]
Could you provide the code snippet in which the error is occurring?

**Reply by wolfbaozi** [reply]
At the beginning, I wanted to download Tailwind   the plugin and complete it in vscode ，But after downloading the plugin, I found that I have been unable to perform this operation for a long time，So I opened the output console and saw
![image](https://github.com/tailwindlabs/tailwindcss/assets/46016243/8a35d3bf-378f-489d-8695-37368107f33d)

**Reply by wolfbaozi** [reply]
I just want to have the code completion function, but I don't know what to do, thanks

**Comment by tlgreg** [comment]
Seems like it's a `boxShadow` value problem in your tailwind config and probably not just an intellisense issue. Eg. if you have a number or a nested object. Should be a string.
Docs: [Box Shadow # Customizing your theme](https://tailwindcss.com/docs/box-shadow#customizing-your-theme)

**Reply by wolfbaozi** [reply]
thanks ,It is indeed caused by this problem。But I don't seem to have any problems with the content here, should I handle it。
![image](https://github.com/tailwindlabs/tailwindcss/assets/46016243/7bfdd7c5-6e33-4324-867e-8239987fcf0f)
![image](https://github.com/tailwindlabs/tailwindcss/assets/46016243/388a539d-0b7d-4304-9b92-79e2815f80ec)

**Reply by tlgreg** [reply, marked as answer]
If this goes into `boxShadow` then the nested object in `shadows-focus` is the issue.
Flattening nested objects is not a general behavior in the theme, it's a feature of `colors`.

**Reply by wolfbaozi** [reply]
Thank you very much. The problem has been resolved
